mmcv/ops/group_points.py

```diff
@@ -210,8 +215,7 @@ def forward(ctx, features: torch.Tensor,
         return output
 
     @staticmethod
-    def backward(ctx,
-                 grad_out: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]:
+    def backward(ctx, grad_out: torch.Tensor) -> Tuple[torch.Tensor, None]:
```

Comment:
Why return type is `Tuple[torch.Tensor, None]`?

Reply:
The return is `grad_features and None`.